LORIS_indepth.html

```diff
@@ -581,85 +295,103 @@ <h2>DICOM archives</h2>
                         <img src="images/DICOM_archive.png">
                       </section>
 		      <section>
-                        <h2>Imaging Uploader -<font color="red"> New</font></h2>
+                        <h2>Imaging Uploader</h2>
                         <img src="images/Imaging_uploader.png">
                       </section>
 				  <section>
-				     <h2>Interactive Protocol Violations -<font color="red"> New</font></h2>
+				     <h2>Interactive Protocol Violations</h2>
 				     <img src="images/Violations.png">
 				   </section>
 
-
 				   <section>
-				    <h2>Genomics Browser -<font color="red"> New</font></h2>
+				    <h2>Genomics Browser</h2>
 				    <img src="images/Imaging_uploader.png">
 				  </section>
 
 				   <section>
-				     <h2>Improved Instrument Builder -<font color="red"> New</font></h2>
+				     <h2>Improved Instrument Builder</h2>
 				     <img src="images/Imaging_uploader.png">
 				   </section>
 				   
 				   <section>
-				     <h2>Configuration module -<font color="red"> New</font></h2>
+				     <h2>Configuration module</h2>
 				     <img src="images/Config.png">
 				   </section>
+            
+	  <section>
+            <h2>Automated PET recognition</h2>
+              <img src="images/PET_GUI.png">
+          </section>
+                    
+	  <section>
+            <h2>Automated PET recognition</h2>
+            <img src="images/PET_Github.png">
+          </section>
+                
+	  <section>
+            <h2>Enhanced DICOM anonymization</h2>
+            <ul>
+          	<li>Going dcmtk free -- PYDICOM</li>
+                <li>CONDA</li>
+                <li>Satra knows everything!!</li>
+            </ul>
+          </section>
+ 
+	  <section>
+            <h2>Enhanced DICOM anonymization</h2>
+            <img src="images/DICOM_Anon.png">
+          </section>
```

Comment:
add to this 2 slides: 
* Electrophysiology Browser for EEG. 
* Automated pipelines for BIDS format data
We'll add EEG browser images to these slides, just make the placeholders for now.

Reply:
done